src/Domain/Model/User.php

```diff
@@ -99,6 +114,18 @@ public function delete(): bool
                 }
             });
 
+        $this->persistences()->get()->each(function ($item) {
+            if (!$item->delete()) {
+                throw new \Exception('Unable to delete persistence records of user');
+            }
+        });
+
+        $this->reminders()->get()->each(function ($item) {
+            if (!$item->delete()) {
+                throw new \Exception('Unable to delete reminder records of user');
+            }
+        });
+
```

Comment:
Should we also delete all throttles here?

Reply:
Will add that in